SU2_IDE/Xcode/SU2_CFD.xcodeproj/project.pbxproj

```diff
@@ -2082,6 +2630,7 @@
 				ENABLE_STRICT_OBJC_MSGSEND = YES;
 				GCC_C_LANGUAGE_STANDARD = gnu99;
 				GCC_NO_COMMON_BLOCKS = YES;
+				GCC_OPTIMIZATION_LEVEL = 0;
```

Comment:
Is debug the default?

Reply:
Hi @pcarruscag ! I removed the optimization because otherwise Xcode will not show the content of the variables during the breakpoints. You will need to print it.... very annoying...

Reply:
I don't think it is a good idea to have different default behavior for different ways to build the code, but then again if someone is using xcode they should know what they are doing...